code/modules/projectiles/ammunition/_ammunition.dm

```diff
@@ -46,6 +46,26 @@
 	icon_state = "[initial(icon_state)][BB ? "-live" : ""]"
 	desc = "[initial(desc)][BB ? "" : " This one is spent."]"
 
+/* 
+ * On accidental consumption, 'spend' the ammo, and add in some gunpowder
+ */
+/obj/item/ammo_casing/on_accidental_consumption(mob/living/carbon/M, mob/living/carbon/user, obj/item/source_item,  discover_after = TRUE)
+	if(BB != null)
+		BB = null
+		update_icon()
+		if(source_item && source_item.reagents)
+			source_item.reagents.add_reagent(/datum/reagent/gunpowder, source_item.reagents.total_volume*(2/3))

```

Comment:
why does swallowing the bullet add gunpowder to the food item it USED to be in?

Reply:
currently it doesn't swallow the bullet (it gets discovered after) I took inspiration from the war movies when they bite the edge of a bullet and use the leaking out gunpower on their wounds or w/e